action.js

```diff
@@ -19,23 +83,141 @@ window.onload = function meunWindow() {
     window.onscroll = function () {};
   }
 
-  open.addEventListener('click', () => {
-    div.style.display = 'block';
+  function addModals() {
+    let modalitem = "";
+    projectsArray.forEach((project, i) => {
+      let tags = "";
+      project.tags.forEach((tag) => {
+        tags += `<span>${tag}</span>`;
+      });
+      modalitem += `
+      <div class="project-item" id="project${i + 1}">
+        <div class="item-header">
+          <div class="title">${project.title}</div>
+          <button data-close-button class="close-button">&times;</button>
+        </div>
+        <div class="item-body">
+          <div class="item-details">
+            <span>${project.company}</span>
+            <span class="item-role">${project.role}</span>
+            <span>${project.year}</span>
+          </div>
+          <div class="item-img img${i + 1}">
+            <p>image point</p>
+          </div>
```

Comment:
- [ ] On the mobile screens the project image is not showing, Please fix the issue and make sure the image is visible on all screens. 

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/162282995-4fda6f2c-aae8-4f51-ba9a-78d54352ad49.png)